src/api/TaskAssignments.js

```diff
@@ -10,9 +10,28 @@ import {
     ERROR_CODE_CREATE_TASK_ASSIGNMENT,
     ERROR_CODE_UPDATE_TASK_ASSIGNMENT,
     ERROR_CODE_DELETE_TASK_ASSIGNMENT,
+    HTTP_STATUS_CODE_RATE_LIMIT,
+    HTTP_STATUS_CODE_INTERNAL_SERVER_ERROR,
+    HTTP_STATUS_CODE_NOT_IMPLEMENTED,
+    HTTP_STATUS_CODE_BAD_GATEWAY,
+    HTTP_STATUS_CODE_SERVICE_UNAVAILABLE,
+    HTTP_STATUS_CODE_GATEWAY_TIMEOUT,
 } from '../constants';
 
+const RETRYABLE = [
+    HTTP_STATUS_CODE_RATE_LIMIT,
+    HTTP_STATUS_CODE_INTERNAL_SERVER_ERROR,
+    HTTP_STATUS_CODE_NOT_IMPLEMENTED,
+    HTTP_STATUS_CODE_BAD_GATEWAY,
+    HTTP_STATUS_CODE_SERVICE_UNAVAILABLE,
+    HTTP_STATUS_CODE_GATEWAY_TIMEOUT,
+];
+
 class TaskAssignments extends Base {
```

Comment:
Can we create a `TasksBase` that handles this logic for all the affected classes?

Reply:
Ok, that is a good way to handle the constants.